client/editEvents.js

```diff
@@ -139,15 +121,12 @@ Template.editEvents.helpers({
   },
 
   rounds: function() {
-    var rounds = Rounds.find({
+    return Rounds.find({
       competitionId: this.competitionId,
       eventCode: this.eventCode
-    }, {
-      sort: {
-        nthRound: 1
-      }
-    }).fetch();
-    return rounds;
+    },
+      { sort: { nthRound: 1 } }
+    ).fetch();
```

Comment:
Looks like this one got a little mangled. Lets move the last } down to line 129 with the )